src/pages/404.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Heading, Stack, VStack, Text } from '@chakra-ui/react'
+import { NextSeo } from 'next-seo'
+import Image from 'next/image'
+import NextLink from 'next/link'
+
+import mascotLogo from '@app-public/mascot-logo.png'
+import { CTAButton } from '~/components/CTAButton'
+import FluidContainer from '~/components/FluidContainer'
+import Layout from '~/components/Layout'
+import Link from '~/components/Link'
+
+const Custom404 = () => (
+  <Layout>
+    <NextSeo title="404 - Not Found" />
+    <Box pb={{ base: 8, lg: 16 }} align="center">
```

Comment:
On the full-width page design on Figma, the mascot picture is 600x600px, to inject a bit of playfullness into the page.

I'm okay with scaling down the image on smaller viewports, but if we have room I'd like the image to take up more space.

We don't have many images anyways, so I'd like to show off those we do have.

What do you think?

Reply:
I mentioned the reason for this in the PR description:

> I've made some changes to the original design, so the banner image is not so big now. This way, the main message about the page wasn't found is readable without scrolling and everything fits better

I'd like to avoid opening a 404 error page just with a huge logo then having to scroll to understand what happened, it's quite confusing.

Reply:
Oh okay, let's keep the focus on the error text instead of on the image then. I think you're probably right that people want to know what went wrong first, instead of spending too much time looking at a picture.